### PR TITLE
Added a function to make a POST request to the chatbot server

### PR DESCRIPTION
**Pull request by Ry-ka**
Mainly boiler plat for the react app side. Added a simple function that takes in a string to create a POST request to the chatbot server and then return the response from the server. You can find that in chatbotApi.ts.

### CODE REVIEW

File: client/chatbotWidget/src/main.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
```
Comment (ErnestoRiDo) [inline client/chatbotWidget/src/main.tsx:4]: Here is where we'll need to add the following line for ant-d: 
import 'antd/dist/reset.css'

**Review by ErnestoRiDo — CHANGES_REQUESTED** [review]
Can you make a new folder labeled components under src and you can just put a blank base.tsx file in there?

File: client/chatbotWidget/src/main.tsx (hunk 1 above)
Comment (Ry-ka) [reply client/chatbotWidget/src/main.tsx:4]: Nice, you can do that when you create the widget.

**Comment by Ry-ka** [comment]
> Can you make a new file labeled components under src and you can just put a blank base.tsx file in there?

You can do that when you create the react component. I just wrote a non-component function that you can import in the components that you create.

**Review by ErnestoRiDo — APPROVED** [review]

**Pull request merged**